viewer/src/datamodels/cad/CadModelUpdateHandler.ts

```diff
@@ -37,18 +38,19 @@ export class CadModelUpdateHandler {
   constructor(sectorRepository: CachedRepository, sectorCuller: SectorCuller) {
     const modelsArray: CadNode[] = [];
     this._updateObservable = combineLatest(
-      this._cameraSubject.pipe(),
+      this._cameraSubject.pipe(auditTime(1000)),
```

Comment:
Why?

Reply:
Camera is the heavy hitter on updates, so we can change that to 700ms as it was. I just thought the others would be nice to have more responsiveness to them.

Reply:
Ok, 700 ms for camera sounds "good"

Reply:
Changed